### ISSUE TITLE
A11y_VisualStudioCodeClient_Welcome Screen_Keyboard: "Hide" button and "More" are not accessible using keyboard 

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#141702]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCode-Win32-Nov2021; #Win32; #Keyboard; #WCAG2.1.1; #AINotInScope;

### Environment Details:
Application Name: Visual Studio Code Client
Windows Version: Windows 11

### Repro Steps:
1)Turn on Narrtor/NVDA
2) Open Visual studio code editor.
3) Using "Alt+H" key navigate to the "Help" menu bar option .
4) Using "Down" arrow key navigate to the "Welcome" item.
5) Make sure you have the following setting in settings.json: "workbench.startupEditor": "gettingStarted"
6) Using "Alt+H" navigate to "Help" menu bar option and use "down" arrow key to navigate to "Getting started".
7) Verify that whether "Hide "and "More" buttons are not accessible using keyboard.

### Actual Result:
 "Hide" button and "More" are not accessible using keyboard .

### Expected Result:
 "Hide" button and "More" should be accessible using keyboard .

### User Impact:
Keyboard users will face difficulty to access the Hide button and more using keyboard.
![A11y_VisualStudioCodeClient_Welcome Screen_Keyboard Hide button and More are not accessible using keyboard](https://user-images.githubusercontent.com/98530069/151544876-caab2f31-a21d-4fb0-ba36-3ef738cbd779.gif)
